admin_ui/src/components/AddRowForm.vue

```diff
@@ -66,7 +68,13 @@ export default {
                     data: json
                 })
             } catch (error) {
-                const data = error.response.data
+                let data = error.response.data
+
+                if (data["db_error"]) {
+                    data = data["db_error"]
+                } else {
+                    data = data["detail"][0]["msg"]
```

Comment:
If there are multiple errors, does this mean we're just showing the first error?

How about we pass in a list of errors to the `RowFormErrors` component instead.

Reply:
Yes. Now we have that situation. We only show one error message at a time, but that makes sense because at that point we only have one status code (422 for db errors or 500 for Pydantic errors) from the server response. Sorry if I am wrong.

Reply:
You're right - there's only one response from the server. But lets say there's a table like this:

```python
class MyTable(Table):
    json_1 = JSONB()
    json_2 = JSONB()
```

And you tried posting back `{'json_1': 123, 'json_2': 456}`. The error message that Pydantic returns will show an error for `json_1` and `json_2`.

Reply:
Here's an example - I triggered two errors in the form (one because the name field is too long, and the other because the JSON is incorrect):

<img width="731" alt="Screenshot 2022-09-07 at 15 23 05" src="https://user-images.githubusercontent.com/350976/188902659-c79db769-c4a9-4f14-a660-a24d22422304.png">

Reply:
You are right. I have to try that, but I can't right now. I'll be able to later tonight.

Reply:
No worries